Src/Fido2/AuthenticatorAttestationResponse.cs

```diff
@@ -145,6 +145,7 @@ public static AuthenticatorAttestationResponse Parse(AuthenticatorAttestationRaw
                 "fido-u2f" => new FidoU2f(),                    // https://www.w3.org/TR/webauthn/#fido-u2f-attestation
                 "packed" => new Packed(),                       // https://www.w3.org/TR/webauthn/#packed-attestation
                 "apple" => new Apple(),                         // https://www.w3.org/TR/webauthn/#apple-anonymous-attestation
+                "apple-appattest" => new AppleAppAttest(),      // https://developer.apple.com/documentation/devicecheck/validating_apps_that_connect_to_your_server  
```

Comment:
This comment is not related to this line, but the user present check on line 118.
Do we know if Apple App Attest provides a user presence check?

Reply:
Doesn't much matter to me, but if it doesn't it's not compliant with [WebAuthn spec](https://w3c.github.io/webauthn/#sctn-registering-a-new-credential), step 13.